### PR TITLE
issues-3: Add menu style

### PR DESCRIPTION
**Pull request by QuentinVecchio**
This PR introduces the menu style. Animation will come in a follow-up PR.

Part of #3 
Tested with #16

Test URLs:
- Before: https://main--durysta--hlxsites.hlx.page
- After: https://3-header--durysta--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -84,30 +73,30 @@ header nav .nav-hamburger-icon::after {
...
-  width: 20px;
+  width: 27px;
 }
 
 header nav .nav-hamburger-icon::before,
 header nav .nav-hamburger-icon::after {
   content: '';
   position: absolute;
-  background: currentcolor;
+  background: #377e0b;
```
Comment (kmurugulla) [inline blocks/header/header.css:83]: would be great if we could use the global variables defined in styles.css

File: blocks/header/header.css [hunk 2]
```diff
@@ -168,14 +204,50 @@ header nav[aria-expanded="true"] .nav-sections {
...
+  padding: 14px 35px;
+  display: block;
+  height: auto;
+  font-size: 18px;
+  line-height: 1.25;
+  color: var(--overlay-color);
+  text-decoration: none;
+  text-align: center;
   font-weight: 700;
+  font-family: 'ITC Avant Garde Gothic W01 Md', var(--body-font-family);
```
Comment (kmurugulla) [inline blocks/header/header.css:231]: duplicate declaration

File: blocks/header/header.css [hunk 3]
```diff
@@ -186,7 +258,27 @@ header nav .nav-sections ul > li > ul > li {
...
+header nav .nav-tools ul li:not(:last-child) {
+  border-right: 1.5px solid #414042 !important;
+}
+
+header nav .nav-sections ul li:not(:last-child):hover {
+  background: #ffc60b;
+}
+
+header nav .nav-sections ul li:not(:last-child):hover a {
+  color: #3a423b;
```
Comment (kmurugulla) [inline blocks/header/header.css:278]: use of variables :) please

**Review by kmurugulla — CHANGES_REQUESTED** [review]

**Comment by kmurugulla** [comment]
@QuentinVecchio  - The header looks great . npm run lint would help with resolving CSS / JS lint issues locally before pushing the code :)

File: blocks/header/header.css (hunk 1 above)
Comment (QuentinVecchio) [reply blocks/header/header.css]: Using global variables now

**Comment by QuentinVecchio** [comment]
> @QuentinVecchio - The header looks great 🥇 . npm run lint would help with resolving CSS / JS lint issues locally before pushing the code

Sorry it was a push before going home operation :)

Now the PR is ready for review and the linter should be good

**Review by kmurugulla — APPROVED** [review]

**Pull request merged**